src/devices/Gpio/Drivers/VirtualGpioPin.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Gpio;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A virtual GpioPin to allow creating a new GpioPin and a new pin number
+    /// </summary>
+    internal class VirtualGpioPin : GpioPin
```

Comment:
Also, do we really need this class? It does not differ from its base class, and isn't even used for type-testing anywhere, it seems.

Reply:
yes because of the constructor. It's the way to call it in a clean way